northstar/src/runtime/console.rs

```diff
@@ -235,7 +125,7 @@ impl Console {
 
     async fn connection<T: AsyncRead + AsyncWrite + Unpin>(
         stream: T,
-        peer: String,
+        peer: ClientId,
         stop: CancellationToken,
         event_tx: EventTx,
         mut notification_rx: broadcast::Receiver<Notification>,
```

Comment:
The `let connect = network_stream.next()` could be selected with `stop` to if the rt shuts down during the connect sequence. Same for `connect.await` etc. This would be a bigger change and probably best would be to refactor the connection sequence into a fn.

Reply:
I agree, probably best to leave this for a next iteration.